### DISCUSSION TITLE
人类一败涂地

### DISCUSSION THREAD

**Discussion by uhyintl** [Gamuxorg/bbs#278] [投递和请求游戏]
游戏名称：人类一败涂地
游戏名称(英文)： Human Fall Flat
下载地址 :（如果有的话）：https://store.steampowered.com/app/477160/__Human_Fall_Flat/

**Comment by shouhuanxiaoji** [comment]
这个游戏的问题是，linux版不再维护了，有也只有旧版本

**Comment by jim486** [comment]
版本已经有更新和steam同步

**Comment by alexliyu7352** [comment]
您网站上没有找到阿?

**Comment by shouhuanxiaoji** [comment]
@alexliyu7352 你指的是网站上没有这个游戏？

**Comment by alexliyu7352** [comment]
是阿, 没找到阿.

小鸡 ***@***.***> 于2021年12月1日周三 下午3:28写道：

**Comment by shouhuanxiaoji** [comment]
处理中

**Comment by shouhuanxiaoji** [comment]
已添加
